packages/flutter/example/pubspec.yaml

```diff
@@ -33,19 +33,19 @@ dependencies:
   #sqflite_common_ffi: ^2.0.0
   #sqflite_common_ffi_web: ^0.3.0
   http: ^1.0.0
-  hive: any  # This gets constrained by `sentry_hive`
+  hive: any # This gets constrained by `sentry_hive`
   sqlite3_flutter_libs: ^0.5.0
 
-dependency_overrides:
-  isar:
-    version: ^3.1.0
-    hosted: https://pub.isar-community.dev/
-  isar_flutter_libs:
-    version: ^3.1.0
-    hosted: https://pub.isar-community.dev/
-  isar_generator:
-    version: ^3.1.0
-    hosted: https://pub.isar-community.dev/
+# dependency_overrides:
+#   isar:
+#     version: ^3.1.0
+#     hosted: https://pub.isar-community.dev/
+#   isar_flutter_libs:
+#     version: ^3.1.0
+#     hosted: https://pub.isar-community.dev/
+#   isar_generator:
+#     version: ^3.1.0
+#     hosted: https://pub.isar-community.dev/
```

Comment:
temporarily comment this out since https://pub.isar-community.dev/ is down currently